pkg/handler/handler.go

```diff
@@ -191,9 +190,12 @@ func (m *Modifier) addEnvToContainer(container *corev1.Container, tokenFilePath,
 	}
 
 	if reservedKeysDefined && regionKeyDefined && regionalStsKeyDefined {
-		return
+		klog.V(3).Infof("container %s has necessary env variables already present",
```

Comment:
I think we might want to set the log level to be different for successes and skips.  I can see in some cases, someone might want to only log successes because the amount of logs generated by skips would be much larger, like 3 for success, 4 or 5 for skip?

Reply:
Good point. I think we should separate them. As per the logging convention doc, I categorized them all as "Extended information about changes" which is V(3). There is only success log per pod, so it makes sense to add them at V(3) and rest as "Debug" info as V(4). Will fix.